src/pudl/metadata/resources/eiaaeo.py

```diff
@@ -0,0 +1,514 @@
+            "sectors used primarily for own-use generation, but which may also "
+            "sell some power to the grid. Solar photovoltaic capacity portion of "
+            "Renewable Sources in gigawatts direct current; other technologies "
+            "in gigawatts alternating current."
+        ),
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_end_use_sector_fuel_type",
+                "net_summer_capacity_mw",  # should this be capacity_mw_eia?
+                "gross_generation_mwh",
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_end_use_sector_fuel_type",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__electricity_sales": {
+        "description": "Projected electricity sales.",
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_sales_sector",
+                "sales_mwh",
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_sales_sector",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__net_energy_for_load": {
+        "description": "Projected net energy for load.",
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_net_energy_for_load_sector",
+                "net_energy_for_load_mwh",
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_net_energy_for_load_sector",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__end_use_prices_by_sector": {
+        "description": "Projected electricity cost to the end user by sector.",
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region",
+                "aeo_case",
+                "projection_date",
+                "aeo_sales_sector",
```

Comment:
If we unstack the res/ind/com/trans stuff above we should unstack it here too. And if not, we shouldn't.